cmd/src/main.go

```diff
@@ -42,6 +42,7 @@ The commands are:
 	extensions,ext  manages extensions (experimental)
 	batch           manages batch changes
 	lsif            manages LSIF data
+	login		authenticate to a Sourcegraph instance with your user credentials
```

Comment:
Although this looks visually aligned on an 8 space tab width, the other lines actually use spaces rather than tabs between the command name and the description (ie between `login` and `authenticate` in this case). Would you be able to change this to match the other lines with spaces, please? (Alternatively, I'm happy to do it and push it to your branch, if you'd prefer!)

Reply:
Hello @LawnGnome 👋 Thanks for having a look! Hmm, I added the line with spaces and double checked the file in VSC just now. It looks like there are spaces versus tabs, but perhaps I'm missing something? 🤔 

<img width="1036" alt="image" src="https://user-images.githubusercontent.com/27694443/159066705-86ee8a6e-60d9-44df-94df-cfa34f79de6c.png">

If I did, could you let me know where the issue is and I am happy to fix it :)

Reply:
This is what I see when I check out your branch:
![image](https://user-images.githubusercontent.com/68532117/159083813-6297efb0-6223-43f8-afa4-0f93abc06b9f.png)
It looks like it's still using 2 tabs instead of 11 spaces.

Reply:
@LawnGnome  Aha! @abeatrix and I figured out the problem. I got turned around in the other branch I had made, and didn't push the changes correctly. 🐝 Helped me sort it out. I also made an addition to the Changelog. Can you have a look and see if its good to go ?